api/data_explorer/controllers/export_url_controller.py

```diff
@@ -1,10 +1,24 @@
 import connexion
 import six
+import json
 
 from flask import current_app
 
 from data_explorer.models.export_url_response import ExportUrlResponse  # noqa: E501
 
 
 def export_url_post():  # noqa: E501
-    return ExportUrlResponse(url='EXPORT_URL')
+    requests = []
+    for table_name in current_app.config['TABLE_NAMES']:
+        # Treat facets as BigQuery tables if they conform to project_id.dataset.table_id
+        if len(table_name.split(".")) == 3:
+            requests.append({
+                "name": current_app.config['DATASET_NAME'],
```

Comment:
```
"name": table_name.split(".")[2],
```

Reply:
Done, for the record I had it this way because in Brad's example the table_id was der7612_deadff but the table_name was nurses_health_study. I guess the table_id is better if we're being data agnostic